core/domain/platform_parameter_domain.py

```diff
@@ -138,12 +138,19 @@ def validate(self):
                 'Invalid browser type \'%s\', must be one of %s.' % (
                     self._browser_type, ALLOWED_BROWSER_TYPES))
 
-        if (
-                self._app_version is not None and
-                APP_VERSION_WITH_HASH_REGEXP.match(self._app_version) is None):
-            raise utils.ValidationError(
-                'Invalid version \'%s\', expected to match regexp %s.' % (
-                    self._app_version, APP_VERSION_WITH_HASH_REGEXP))
+        if self._app_version is not None:
+            match = APP_VERSION_WITH_HASH_REGEXP.match(self._app_version)
+            if match is None:
+                raise utils.ValidationError(
+                    'Invalid version \'%s\', expected to match regexp %s.' % (
```

Comment:
Do you have a test for this case, too?

Reply:
Yes, and the backend tests already makes sure all code paths are covered.